database-plugins/src/main/java/co/cask/hydrator/plugin/db/batch/action/DBConfig.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.hydrator.plugin.db.batch.action;
+
+import co.cask.cdap.api.annotation.Description;
+import co.cask.cdap.api.annotation.Macro;
+import co.cask.hydrator.plugin.ConnectionConfig;
+import co.cask.hydrator.plugin.DBManager;
+import org.slf4j.Logger;
+
+import java.sql.Connection;
+import java.sql.Driver;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+/**
+ * Config for Actions running database commands
+ */
+public class DBConfig extends ConnectionConfig {
+
+  public static final String JDBC_PLUGIN_ID = "driver";
+
+  @Description("The database command to run.")
+  @Macro
+  public String command;
+
+  public DBConfig() {
+    super();
+  }
+
+  public void executeQuery(Class<? extends Driver> driverClass, Logger log) {
```

Comment:
create a new class rather than put this in the config, as it doesn't really have anything to do with configuration.